### PR TITLE
Changing pilot areas

### PR DESCRIPTION
**Pull request by andrewsprinz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ The app is currently hosted on `heroku`, with the following versions:
 
 Heroku is set up to automatically deploy from `staging` and `master` branches (to staging and production versions respectively).
 
+You may have to run migrations manually:
+
+``heroku run "mix ecto.migrate" --app action-for-children-staging``
```
Comment (andrewMacmurray) [inline README.md]: Did you try adding

```
release: MIX_ENV=prod mix ecto.migrate
```

To the `Procfile`?

Comment (andrewsprinz) [reply README.md]: Not tried yet, but will give it a shot!

Comment (andrewMacmurray) [reply README.md]: Cool, let me know when you've tried it, otherwise can merge in

Comment (andrewsprinz) [reply README.md]: Added it in, see how it goes!

Comment (andrewMacmurray) [reply README.md]: Great, you'll want to remove these lines from the README in that case

**Pull request merged**